container_registry/container_analysis/README.md

```diff
@@ -0,0 +1,59 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google
+Cloud Platform logo" title="Google Cloud Platform" align="right" height="96"
+width="96"/>
+
+# Google Cloud Container Analysis Samples
+
+
+Container Analysis scans container images stored in Container Registry for vulnerabilities.
+Continuous automated analysis of containers keep you informed about known vulnerabilities so 
+that you can review and address issues before deployment.
+
+Additionally, third-party metadata providers can use Container Analysis to store and 
+retrieve additional metadata for their customers' images, such as packages installed in an image.
+
+
+## Description
+
+These samples show how to use the [Google Cloud Container Analysis Client Library](https://cloud.google.com/container-registry/docs/reference/libraries).
+
+## Build and Run
+1.  **Enable APIs** - [Enable the Container Analysis API](https://console.cloud.google.com/flows/enableapi?apiid=containeranalysis.googleapis.com)
+    and create a new project or select an existing project.
+1.  **Install and Initialize Cloud SDK**
+    Follow instructions from the available [quickstarts](https://cloud.google.com/sdk/docs/quickstarts)
+1.  **Clone the repo** and cd into this directory
+
+```
+    $ git clone https://github.com/GoogleCloudPlatform/python-docs-samples
+    $ cd python-docs-samples/container-registry/container-analysis
+```
+
+1. **Install Dependencies**
+
+```
+    $ python3 -m venv ./venv
+    $ source ./venv/bin/activate
+    $ pip install -r requirements.txt
+```
+
+1. **Set Environment Variables**
+
+```
+    $ export GOOGLE_CLOUD_PROJECT="YOUR_PROJECT_ID"
```

Comment:
GOOGLE_CLOUD_PROJECT -> GCLOUD_PROJECT

Reply:
Should users also set their application credentials here?

Reply:
Is that required? I don't have it set in my terminal and it's working, I assume because I authenticated locally with "gcloud login"

I'm not really sure how much detail is required for set up here, so I copied this readme from a different project